Makefile

```
@@ -683,6 +684,20 @@ test-capd-junit: $(SETUP_ENVTEST) $(GOTESTSUM) ## Run unit and integration tests
 	$(GOTESTSUM) --junitfile $(ARTIFACTS)/junit.infra_docker.xml --raw-command cat $(ARTIFACTS)/junit.infra_docker.stdout
 	exit $$(cat $(ARTIFACTS)/junit.infra_docker.exitcode)
 
+.PHONY: test-test-extension
```

```suggestion
.PHONY: test-extension
```
I will avoid the repetition, we have only the test extensions in CAPI
same for the other tasks

---

No objections 😀

---

This is not a good solution IMO as it makes it very confusing - this looks like it will build the test extension - e.g. `make clusterctl`

---

Good point